test/t/ngx_mruby.rb

```diff
@@ -512,6 +512,12 @@ def is_async_supported?
   t.assert_equal "", `#{cmd_h}`.chomp
 end
 
+t.assert('ngx_mruby - validate client certificate without validation error') do
+  path = ENV['NGINX_INSTALL_DIR'] + '/html'
+  res = `openssl s_client -servername localhost -cert #{path}/client.crt -key #{path}/client.key -CAfile localhost.crt -connect localhost:58072 < /dev/null 2>&1 > /dev/null | grep 'verify error' | awk -F'[:]' -F'[=]' '{print $2}'`
```

Comment:
> -CAfile localhost.crt 

@Zogoo We don't have  localhost.crt in the working directory.  Is it correct path for CA file.

Reply:
Here are results of openssl command.

WIth existing CA file.

```
% openssl s_client -servername localhost -cert ./build/nginx/html/client.crt -key ./build/nginx/html/client.key -CAfile ./build/nginx/html/localhost.crt -connect localhost:58072 < /dev/null 2>&1 > /dev/null
depth=0 C = XX, L = Default City, O = Default Company Ltd
verify return:1
DONE
```

With non existing CA file.

```
% openssl s_client -servername localhost -cert ./build/nginx/html/client.crt -key ./build/nginx/html/client.key -CAfile localhost.crt -connect localhost:58072 < /dev/null 2>&1 > /dev/null
140478988285840:error:02001002:system library:fopen:No such file or directory:bss_file.c:175:fopen('localhost.crt','r')
140478988285840:error:2006D080:BIO routines:BIO_new_file:no such file:bss_file.c:182:
140478988285840:error:0B084002:x509 certificate routines:X509_load_cert_crl_file:system lib:by_file.c:258:
depth=0 C = XX, L = Default City, O = Default Company Ltd
verify error:num=18:self signed certificate
verify return:1
depth=0 C = XX, L = Default City, O = Default Company Ltd
verify return:1
DONE
```

Reply:
Thanks for your help I added the path (#{path}) for CA file.